salt/orch/kubernetes.sls

```diff
@@ -127,6 +127,7 @@ reboot_setup:
     - require:
       - salt: kube_master_setup
 
+{% if salt['mine.get']('roles:kube-(master|minion)', 'caasp_fqdn', expr_form='grain_pcre').values()|length > 0 %}
```

Comment:
I am not sure this is needed - if there was no machines with these roles, the rest of the orch would have failed as well.